libs/article/api/src/lib/article.service.ts

```diff
@@ -630,7 +638,8 @@ const createDraftFilter = (filter: Partial<ArticleFilter>): Prisma.ArticleWhereI
     return {
       revisions: {
         some: {
-          publishedAt: filter.draft ? null : {not: null}
+          publishedAt: filter.draft ? null : {not: null},
+          archivedAt: null
```

Comment:
should be set to archivedAt null even when draft is false, as only draft can be archived.